File: jaynetwork/src/main/java/com/jaycee/jaynetwork/manager/listener/NetworkListener.java
package com.jaycee.jaynetwork.manager.listener;

import com.jaycee.jaynetwork.manager.constants.ConnectivityType;

public interface NetworkListener {
    void isConnected(boolean status);

    void networkType(ConnectivityType connectivityType);

    void isConnectionFast(boolean isFast);
}

File: jaynetwork/src/main/java/com/jaycee/jaynetwork/manager/listener/NetworkListener.kt
package com.jaycee.jaynetwork.manager.listener

import com.jaycee.jaynetwork.manager.constants.ConnectivityType

interface NetworkListener {
    fun isConnected(status: Boolean)
    fun networkType(connectivityType: ConnectivityType?)
    fun isConnectionFast(isFast: Boolean)
}